core/src/plugins/filed/python/module/bareosfd.h

```diff
@@ -687,6 +687,7 @@ MOD_INIT(bareosfd)
   ConstSet_StrLong(pDictbVariable, bVarPrevJobName, 18);
   ConstSet_StrLong(pDictbVariable, bVarPrefixLinks, 19);
   ConstSet_StrLong(pDictbVariable, bVarCheckChanges, 20);
+  ConstSet_StrLong(pDictbVariable, bVarUsedConfig, 21);
```

Comment:
Do you know why we use the numerical constants here instead of the defines directly,
i.e. `ConstSet_StrLong(pDictbVariable, bVarUsedConfig, bVarUsedConfig)` ?

I thought it was done this way to make the python api more stable (since the numbers wont change even if the enum changes), but in that case the `GetValue()`/`SetValue()` calls would still need to be updated anyways.
Furthermore python plugins shouldnt even touch the numerical constants anyways and just use the values we define here so it shouldnt even be an issue if we were to (accidentally) change them!

Reply:
I don't know, I guess it is one of these "it has always been that way" things. I guess it is a good time to change that.